packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -62,12 +62,22 @@ export class Loan {
   /**
    * List all available loan schemes.
    *
-   * @return {Promise<LoanSchemeResult[]>}
+   * @return {Promise<ListLoanSchemeResult[]>}
    */
-  async listLoanSchemes (): Promise<LoanSchemeResult[]> {
+  async listLoanSchemes (): Promise<ListLoanSchemeResult[]> {
     return await this.client.call('listloanschemes', [], 'bignumber')
   }
 
+  /**
+   * Get loan scheme.
+   *
+   * @param {string} id Unique identifier of the loan scheme, max 8 chars
```

Comment:
the max 8 chars is enforced on creation, right ? 
It is confusing here.

Reply:
Don't think it is confusing. This is the same description for id across loan rpcs

Reply:
If you request an id with more than 8 chars, it won't raise any error. It just won't find any loan scheme.

Reply:
Hmm, I shall inform C++ programmer to fix this.